docs/source/redhat-opens-install.rst

```diff
@@ -59,6 +59,10 @@ Instructions for installing OpenStack on RHEL can be found here --
 Configure YUM repositories
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+If you're using CentOS, check the yum priorities plugin is installed:
+
+    yum install yum-plugin-priorities
```

Comment:
This doesn't actually render as a monospaced block; see http://calico-jenkins/pr-docs/centos-install/redhat-opens-install.html. You need to double up the colon on the previous line.